src/modules/user/domain/events/events-success-domain/user-destroyed.event.ts

```diff
@@ -0,0 +1,6 @@
+export class UserDestroyedEvent {
+  constructor(
+    public readonly uuid: string,
+    public readonly deletedAt: Date,
```

Comment:
Manéjalo como destroyedAt, si esto fue de un copy paste tambien cambiarlo en el otro de favor
```suggestion
    public readonly destroyedAt: Date,
```

Reply:
@2314-RAustin Listo, corregido. 

También se cambió en el evento destroy de Role.